design-system/tokens.css

```
@@ -0,0 +1,76 @@
+  --space-1: 4px;  --space-2: 8px;  --space-3: 12px;  --space-4: 16px;  --space-6: 24px;
+  --card-pad: 18px;
+  --control-pad-y: 8px;  --control-pad-x: 10px;
+  --row-pad-y: 9px;      --row-pad-x: 10px;
+  --fs-base: 14px;  --fs-sm: 12.5px;  --fs-xs: 11.5px;
+  --kpi-size: 26px;
+
+  /* ---- shape (a theme may override) ---- */
+  --radius: 10px;  --radius-sm: 8px;  --btn-radius: 8px;  --input-radius: 8px;
+
+  /* ---- type ---- */
+  --font-body: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
```

Stylelint (17.13.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)